content/en/logs/log_collection/kotlin-multiplatform.md

```diff
@@ -55,6 +55,7 @@ kotlin {
     | 0.0.1                                    | 2.14.1                  |
     | 0.0.2                                    | 2.17.0                  |
     | 0.0.3                                    | 2.17.0                  |
+    | 0.4.0                                    | 2.20.0                  |
```

Comment:
Should this be `0.0.4`, which more closely matches the ones above it?

Reply:
no, this is a correct change, see https://github.com/DataDog/dd-sdk-kotlin-multiplatform/tags

Reply:
Great, just wanted to double-check!